src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.Unix.cs

```diff
@@ -2,29 +2,32 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 using System.Diagnostics;
 using System.Text;
 
 namespace System.Runtime.InteropServices
 {
     public static partial class Marshal
     {
-        public static string PtrToStringAuto(IntPtr ptr, int len)
+        // TODO-NULLABLE: This has different behavior from the other PtrToString(IntPtr, int) functions
+        //                This is due to PtrToStringUTF8 on Unix and is being resolved independently
```

Comment:
Do we still need the TODO-NULLABLE then?  Or can you include the issue #?

Reply:
I could probably update the comment, but it likely isn't worth the extra CI cycles. I'm just waiting on https://github.com/dotnet/coreclr/pull/23731 to be merged and for us to pull it back into NullableFeature (probably resolved Monday).